public/app/percona/dbaas/components/DBCluster/AddDBClusterModal/DBClusterAdvancedOptions/DBClusterAdvancedOptions.tsx

```diff
@@ -149,6 +149,12 @@ export const DBClusterAdvancedOptions: FC<FormRenderProps> = ({ values, form })
     return () => clearTimeout(expectedTimer);
   }, [memory, cpu, disk, kubernetesCluster, topology, nodes, single, databaseType]);
 
+  useEffect(() => {
+    if (topology === DBClusterTopology.cluster && nodes < MIN_NODES) {
+      change(AddDBClusterFields.nodes, MIN_NODES);
+    }
+  }, [topology]);
```

Comment:
isn't `nodes` a dependency as well?

Reply:
No, this is meant to run only when topology changes. You can change the nodes manually and we don't want to force it to the min while you are typing or clicking the arrows because a validation message will appear for such cases. We only want to set it to the proper value when the user changes topology.